best_practices/forms.rst

```diff
@@ -207,3 +207,21 @@ Second, we recommend using ``$form->isSubmitted()`` in the ``if`` statement
 for clarity. This isn't technically needed, since ``isValid()`` first calls
 ``isSubmitted()``. But without this, the flow doesn't read well as it *looks*
 like the form is *always* processed (even on the GET request).
+
+Custom Form Field Types
+-----------------------
+
+.. best-practice::
+
+    Add the ``app.`` prefix to your custom form field types to avoid collisions.
```

Comment:
dots are forbidden in type names, so your suggestion is really bad. Recommend ``app_`` instead.

for reference, type names are validated as ``/^[a-z0-9_]*$/i``